Tools/autotest/autotest.py

```diff
@@ -131,7 +132,7 @@ def build_devrelease():
 
 def build_examples():
     """Build examples."""
-    for target in 'px4-v2', 'navio':
+    for target in 'fmuv2', 'px4-v2', 'navio', 'linux':
```

Comment:
Do we really want this many targets? Giving that we wont run most of them this primarily seems like a waste of processing time. (IE I'd probably keep it to just Linux, or allow the `build_examples()`  to take a list of what targets we actually want to run.)